### PR TITLE
Add a grid based schedule

### PR DESCRIPTION
**Pull request by AdamBark**
Use css display: grid with a 5 minute row size

### CODE REVIEW

**Comment by AdamBark** [comment]
I didn't just push this straight to main because of a couple of issues in homepage.yml. There are two talks scheduled for the same five minute slot?! and the last two events have no venue assigned.

File: data/homepage.yml [hunk 1]
```diff
@@ -76,129 +76,229 @@ tab:
...
     - speaker : "Rajeshkumar Sampathrajan"
       speaker_image : "images/speakers/rajeshkumar-sampathrajan-thumb.jpg"
       subject : "Be Up for PCA success"
       venue : "Room 4"
+      column: 5
       time : "4:15 PM - 4:20 PM"
+      start_hour: 16
+      start_minute: 15
+      end_hour: 16
+      end_minute: 20
```
Comment (AdamBark) [inline data/homepage.yml]: These two overlap and are two short to render properly. Will there be talks this short?

Comment (carlwgeorge) [reply data/homepage.yml]: That's my mistake, the second one should be 4:20 PM to 4:25 PM, as per the [KCD schedule](https://community.cncf.io/events/details/cncf-kcd-texas-presents-kcd-texas-2024/).  As far as I know the duration is correct.  If rendering them is an issue, what about lumping them together as "KCD lightning talks" from 4:15 PM to 4:25 PM?

**Comment by carlwgeorge** [comment]
I think we'll do the raffle and lightning talks in the same area as the TXLF keynote, so Exhibit Hall 2.  Would you like me to go ahead and fix that in main and then you can rebase the PR?

File: data/homepage.yml (hunk 1 above)
Comment (AdamBark) [reply data/homepage.yml]: Rendering them isn't an issue but having enough space to fit even a ten-minute slot in takes a lot of room for the whole schedule. Are we going to have abstracts to link to at some point? Do you think we should just have a huge schedule?

Comment (carlwgeorge) [reply data/homepage.yml]: I double checked the KCD schedule, and they have a TBD slot from 4:25 PM to 4:30 PM.  I checked out this branch locally and replaced the short sessions with one "KCD lightning talks" session from 4:15 PM to 4:30 PM.  It renders well enough to read the title, but you have to scroll for the time.

Screenshot from 2024-03-13 11-19-26

The abstracts are all in PaperCall.  I do think we should figure out a place to put them on the site, but I don't know where that would be.

**Comment by AdamBark** [comment]
I tried setting a min-width on the schedule and I think it should work OK just requiring horizontal scrolling on a narrow screen, do you think that is reasonable?

**Comment by carlwgeorge** [comment]
I took a look at that, but I don't like how it results in a mix of a mobile menu and header and a wider middle area.

Screenshot 2024-04-04 at 17-27-53 Schedule

Instead of shoehorning the larger grid into the mobile-friendly site layout, what about creating a separate page for the grid view?  I did something similar for the venue map.  The [map page](https://2024.texaslinuxfest.org/map/) has an image of the map that looks fine on desktop, but is pretty small on mobile.  So I made the image a link to open the [image full size](https://2024.texaslinuxfest.org/images/map.png), and added a "click to expand" note in the page text.  The main site is all mobile friendly with no horizontal scroll, but the full size image is loaded it can be zoomed and scrolled any direction.  If we create a standalone page with no mobile features that just has the grid schedule (no headers or footers), on the schedule page we can have the text "click here for an alternate grid view".

**Comment by AdamBark** [comment]
Thanks, what do you think to this? There's a full page grid per day in the Schedule menu

**Pull request closed without merging**